client/containers/DashboardSettings/CommunitySettings.tsx

```diff
@@ -222,6 +228,65 @@ const CommunitySettings = () => {
 						setDescription(evt.target.value.substring(0, 280).replace(/\n/g, ' '));
 					}}
 				/>
+				<InputField
+					label={
+						<span>
+							Journal Citation
+							<Tooltip
+								content={
+									<span>
+										When filled out, this field will be used as the journal
+										title for
+										<br />
+										issue collections in citations, PDF exports, and Crossref
+										deposits.
+										<br />
+										By default, the Community Title is used.
+									</span>
+								}
+								// @ts-expect-error ts-migrate(2322) FIXME: Type '{ children: Element; content: Element; toolt... Remove this comment to see the full error message
+								tooltipClassName="bp3-dark"
```

Comment:
I think we might be able to remove these `tooltipClassName` props (and the corresponding `// @ts-expect-error` comments) while we're in this file. I removed them in a local branch and the tooltips seem to render exactly the same.